database/API.py

```diff
@@ -19,4 +19,136 @@
 
 
 engine = get_azure_engine()
-session = Session(engine)
\ No newline at end of file
+session = Session(engine)
+
+@app.route('/features/get-all', methods=['GET'])
```

Comment:
This seems to be adding some code that's already added by previous PRs. It might be a git issue? Please correct me if I am wrong

Reply:
Yup, Elijah - close this PR. Do not merge it.